Sources/EduIDExpansion/Flows/Home/HomeViewControllerDelegate.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+protocol HomeViewControllerDelegate: AnyObject {
```

Comment:
Did you overlook to change this delegate? It follows a different pattern from the others: `sender: AnyObject` vs. `viewController: UIViewController`.

BTW, not sure why you type erase the view controller anyway. `viewController: HomeViewController` would work too.

Reply:
I get it, no it was not purposefully. Maybe I was scared of the strong contract between the delegate and the viewcontroller

Reply:
I did this